src/interfaces/wallet.cpp

```diff
@@ -202,6 +202,12 @@ class WalletImpl : public Wallet
         LOCK(m_wallet->cs_wallet);
         return m_wallet->ListLockedCoins(outputs);
     }
+    bool IsUsedDestination(const CTxDestination& dst) override
```

Comment:
There was some discussion elsewhere which, IIRC, say you should use `isUsedDestination` instead.

Reply:
yes, makes sense as it follows the same convention as the rest of the file. will update along with other nits

Reply:
fixed